frontend/components/home/exampleData/userData.js

```diff
@@ -61,8 +61,8 @@ export const userData = [
     username: 'Cocopuff',
     dog_name: 'Coco',
     breed: 'Pomeranian',
-    size: 'small',
-    energy: 'high',
+    size: 'Small',
+    energy: 'High',
```

Comment:
Heads up that this data will be coming from the server all lowercase

Reply:
Noted! Thank you for the heads up Kate. I'll make the appropriate styling changes on the front end.